koku/api/report/queries.py

```diff
@@ -647,6 +647,7 @@ def _ranked_list(self, data_list):
             group_by = self._get_group_by()
             for group in group_by:
                 other[group] = 'Other'
+            if 'account' in group_by:
```

Comment:
This doesn't look like it's indented correctly. I think you want the if-statement inside the for-loop.